packages/js/product-editor/src/components/pricing-block/edit.tsx

```diff
@@ -0,0 +1,91 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { createElement, useContext, Fragment } from '@wordpress/element';
+import interpolateComponents from '@automattic/interpolate-components';
+import { Link } from '@woocommerce/components';
+import { useBlockProps } from '@wordpress/block-editor';
+import { useEntityProp } from '@wordpress/core-data';
+import { BlockAttributes } from '@wordpress/blocks';
+import { CurrencyContext } from '@woocommerce/currency';
+import { getSetting } from '@woocommerce/settings';
+import { recordEvent } from '@woocommerce/tracks';
+import {
+	BaseControl,
+	// @ts-expect-error `__experimentalInputControl` does exist.
+	__experimentalInputControl as InputControl,
+} from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import { formatCurrencyDisplayValue } from '../../utils';
+import { useCurrencyInputProps } from '../../hooks/use-currency-input-props';
+
+export function Edit( { attributes }: { attributes: BlockAttributes } ) {
+	const blockProps = useBlockProps();
+	const { name, label, showPricingSection = false } = attributes;
+	const [ regularPrice, setRegularPrice ] = useEntityProp< string >(
+		'postType',
+		'product',
+		name
+	);
+	const context = useContext( CurrencyContext );
+	const { getCurrencyConfig, formatAmount } = context;
+	const currencyConfig = getCurrencyConfig();
+	const inputProps = useCurrencyInputProps( {
+		value: regularPrice,
+		setValue: setRegularPrice,
+	} );
+
+	const taxSettingsElement = showPricingSection
+		? interpolateComponents( {
+				mixedString: __(
+					'Manage more settings in {{link}}Pricing.{{/link}}',
+					'woocommerce'
+				),
+				components: {
+					link: (
+						<Link
+							href={ `${ getSetting(
+								'adminUrl'
+							) }admin.php?page=wc-settings&tab=tax` }
+							target="_blank"
+							type="external"
+							onClick={ () => {
+								recordEvent(
+									'product_pricing_list_price_help_tax_settings_click'
+								);
+							} }
+						>
+							<></>
+						</Link>
+					),
+				},
+		  } )
+		: null;
+
+	return (
+		<div { ...blockProps }>
+			<BaseControl id={ 'product_pricing_' + name } help={ '' }>
+				<InputControl
+					name={ name }
+					onChange={ setRegularPrice }
+					label={ label || __( 'Price', 'woocommerce' ) }
+					value={ formatCurrencyDisplayValue(
+						String( regularPrice ),
+						currencyConfig,
+						formatAmount
+					) }
+					{ ...inputProps }
+				/>
+			</BaseControl>
+			{ taxSettingsElement && (
+				<span className="woocommerce-product-form__secondary-text">
+					{ taxSettingsElement }
```

Comment:
Why you are not using the `help` prop of the `BaseControl` component for this case?

Reply:
Good point! Fixed here: https://github.com/woocommerce/woocommerce/pull/37211/commits/dfe22e54ed1b183a095ae9d053c265daac791824